src/main.c

```diff
@@ -2234,6 +2236,40 @@ static int on_config_error_log(h2o_configurator_command_t *cmd, h2o_configurator
     return 0;
 }
 
+static int on_config_h2olog_socket(h2o_configurator_command_t *cmd, h2o_configurator_context_t *ctx, yoml_t *node)
+{
+    yoml_t **path_node, **owner_node, **group_node, **permission_node, **sndbuf_node, **appdata_node;
+    if (h2o_configurator_parse_mapping(cmd, node, "path:s", "owner:s,group:s,permission:s,sndbuf:s,appdata:s", &path_node, &owner_node,
+                                       &group_node, &permission_node, &sndbuf_node, &appdata_node) != 0)
+        return -1;
+
+    const char *path = (*path_node)->data.scalar;
+    struct sockaddr_un sa;
+    if (strlen(path) >= sizeof(sa.sun_path)) {
+        h2o_configurator_errprintf(cmd, node, "path:%s is too long as a unix socket name", path);
+        return -1;
+    }
+    sa = (struct sockaddr_un){
+        .sun_family = AF_UNIX,
+    };
+    strcpy(sa.sun_path, path);
+
+    int fd = open_unix_listener(cmd, node, &sa, owner_node, group_node, permission_node);
```

Comment:
I'm not sure if we want to abuse for accepting h2olog connections.

`listener_config_t` is designed specifically for accepting HTTP connections, it has things like `hostconf` and QPACK settings. We do not want to stop accepting h2olog clients when we hit `num-connections`.

How about spawning a dedicated thread for accepting h2olog connections right before spawning worker threads in `main`?

Reply:
fd4bb05063754bb6756db301779b756421eb3cd0 done

Agreed that h2olog connections should be handled outside of the HTTP workers, even though it could deal with HTTP in the future.